app/code/Magento/Customer/Test/Mftf/Test/StorefrontVerifyCustomCustomerAddressAttributeTest.xml

```diff
@@ -0,0 +1,144 @@
+
+        <!--Add Second Shipping Address-->
+        <click selector="{{CheckoutShippingSection.newAddressButton}}" stepKey="ClickOnAddNewAddressButton"/>
+        <waitForPageLoad stepKey="waitForNewAddressPageToLoad"/>
+
+        <!--Fill in Shipping Address required fields and Custom Customer Address Attribute and click *Ship Here* button-->
+        <actionGroup ref="FillNewShippingAddressModalActionGroup" stepKey="changeAddress">
+            <argument name="address" value="US_Address_NY"/>
+        </actionGroup>
+        <actionGroup ref="StorefrontSelectDropdownCustomerAddressAttributeValueActionGroup" stepKey="selectOptionValue2">
+            <argument name="customerAddressAttribute" value="AttributeDropdownData"/>
+            <argument name="optionValue" value="{{AttributeDropdownData.firstOption}}"/>
+        </actionGroup>
+        <click selector="{{CheckoutShippingSection.saveAddress}}" stepKey="saveNewAddress"/>
+
+        <!-- Ensure that the selected shipping address is similar to second address -->
+        <grabTextFrom selector="{{CheckoutShippingSection.selectedShippingAddress}}" stepKey="grabSelectedShippingAddress1"/>
+        <assertStringContainsString stepKey="assertShippingAddressStreet1">
+            <expectedResult type="string">{{US_Address_NY.street[0]}}</expectedResult>
+            <actualResult type="variable">$grabSelectedShippingAddress1</actualResult>
+        </assertStringContainsString>
+        <!--Verify that selected "Test Dropdown" options appears on the page-->
+        <assertStringContainsString stepKey="assertShippingAddressCustomAttribute1">
+            <expectedResult type="string">{{AttributeDropdownData.firstOption}}</expectedResult>
+            <actualResult type="variable">$grabSelectedShippingAddress1</actualResult>
+        </assertStringContainsString>
+
+        <!-- Select First Shipping Method and Go To Billing Section -->
+        <waitForLoadingMaskToDisappear stepKey="waitForLoadingMask"/>
+        <click selector="{{CheckoutShippingSection.firstShippingMethod}}" stepKey="selectFirstShippingMethod"/>
+        <waitForElement selector="{{CheckoutShippingSection.next}}" time="30" stepKey="waitForNextButton"/>
+        <click selector="{{CheckoutShippingSection.next}}" stepKey="clickNext"/>
+
+        <!-- Ensure that the Billing address is similar to first address -->
+        <grabTextFrom selector="{{CheckoutBillingAddressSection.selectedBillingAddress}}" stepKey="grabSelectedBillingAddress"/>
+        <assertStringContainsString stepKey="assertBillingAddressStreet">
+            <expectedResult type="string">{{CustomerAddressSimple.street[0]}}</expectedResult>
+            <actualResult type="variable">$grabSelectedBillingAddress</actualResult>
+        </assertStringContainsString>
+
+        <!--Verify that selected "Test Dropdown" options appears on the page-->
+        <assertStringContainsString stepKey="assertBillingAddressCustomAttribute">
```

Comment:
change to `change to <see selector="{{CheckoutBillingAddressSection. selectedBillingAddress}}" userInput="{{AttributeDropdownData.firstOption}}" stepKey="seeBillingAddressCustomAttribute"/>`